config/initializers/devise.rb

```diff
@@ -317,4 +317,17 @@
         scope: 'userinfo.email,userinfo.profile,calendar'
       }
     )
+
+    config.omniauth(
+      :facebook,
+      ENV['FACEBOOK_APP_ID'],
+      ENV['FACEBOOK_APP_SECRET'],
+      scope: 'public_profile,email',
+      info_fields: 'email,first_name,last_name',
```

Comment:
Just double-checking here, should this be a string or an array?

Reply:
From the [Devise documentation](https://github.com/plataformatec/devise/wiki/OmniAuth:-Overview), it should be a string.